examples/all-clusters-app/nrfconnect/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,27 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# Multirole is the only currently supported role by SoftDevice.
+CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=y
+
+# ZMS cache optimization
+CONFIG_ZMS_LOOKUP_CACHE=y
+CONFIG_ZMS_LOOKUP_CACHE_SIZE=512
+CONFIG_ZMS_LOOKUP_CACHE_FOR_SETTINGS=y
+
+# Workaround required as Zephyr L2 implies usage of NVS backend for settings.
+# It should be removed once the proper fix will be applied in Zephyr.
+CONFIG_NVS=n
```

Comment:
Do we still need this workaround? Also, the ZMS config above? We can use the defaults for that.

Reply:
Please note this is based on NCS 3.0.0, so I have used configuration that was used at that time